Makefile

```
@@ -52,12 +52,18 @@ val: all
 check:
 	@cppcheck --enable=all $$(find . -type f -name "*.cpp" | tr '\n' ' ')
 
-#--------------------------------------------
-# for test
-#--------------------------------------------
-.PHONY	: req
-req:
-	echo -n "Hello from client" | nc 127.0.0.1 8080
+.PHONY	: test
+test:
+	@make unit && make e2e
+
+.PHONY	: unit
+unit:
+	-@pytest ./test/unit
+
+.PHONY	: e2e
+e2e:
+	-@pytest ./test/integration
```

`make unit` と `make e2e` の失敗を `-` で消すと、actions で `make test` が呼ばれた時に、 pytest 自体に失敗しても `make test` の exit status は 0 になって actions が常に成功と判定しそうなので、ここでは `-` はなくした方が良いと思うのですが、どうでしょう
試しに `test_webserv()` の中に exit(1) と書いて `make test` すると、pytest の失敗は出力されますが、`echo $?` の結果が 0 になってしまいます
```suggestion
.PHONY	: unit
unit:
	@pytest ./test/unit

.PHONY	: e2e
e2e:
	@pytest ./test/integration
```

---

バグ見つけてくださってありがとうございます！